src/libs/actions/Link.ts

```diff
@@ -10,29 +8,23 @@ import * as Url from '../Url';
 let isNetworkOffline = false;
```

Comment:
```suggestion
let isNetworkOffline: boolean | undefined = false;
```

Same.

Reply:
actually here it would stay `false` before this callback fires 🤔

Reply:
You are right @adhorodyski! Sorry @kubabutkiewicz , but could you revert this one?